### ISSUE TITLE
Provide PCB Source Design Files

### ISSUE THREAD

**Issue by JoNilsson** [ztto/ArdSCSino-stm32#7]
Would like to modify the design to include: 

• 4-way Berg type power connector to allow the onboard AMS1117-3.3 Voltage Regulator of the STM32 Bluepill to be powered from host systems 4.75 to 12 VDC for easy integration into vintage computers.   
  
• Headers for placing termination resistor arrays.  
  
Will you provide the design files?

**Comment by buserror** [comment]
I've just redone a base design for a bluepill <-> SD <-> SCSI board in kicad this afternoon. Its likely not perfect, and I did try to make it compact (as it's considerably cheaper) -- I plan to push the design on my github soon, possibly as soon as I get the source compiled. https://oshpark.com/shared_projects/6McULRJk $21 for 3.

**Comment by DOSputin** [comment]
> I've just redone a base design for a bluepill <-> SD <-> SCSI board in kicad this afternoon.  

 So just to be clear.  You are talking about a redesign of this board, or does it utilize @ztto's source files?  
Either way,  would be very interested. Just followed you, will keep any eye out for the push.  Also, if you need any help let me know, would be happy to lend my proficiency with KiCad to the effort.   while Oshpark is great, but I would suggest [JLCPCB.com,](https://www.jlcpcb.com) they offer PCB and SMT assembly (if you use their parts library)  Their quality is excellent, and prices are very reasonable.

**Comment by buserror** [comment]
There are only gerbers on this repo, so I took the original PNG and re-did a schematic for it. I added a bit of decoupling and some optional pullups for the SD card. Also, I used a micro-sd card I can source (molex, which is in Kikad's lib). I thought of make the board bigger to match 3.5" mounting holes, but it's a ridiculous (and expensive) waste of board space, so I kept 2 mounting holes so the board can be mounted on a strip for example.
Since then I discovered the PNG has one pin wrong, so I re-posted a new version at oshpark -- thanks for the tip of jlcpcb -- they seem quite competitive indeed!
I'll slap together a repo and push what I've done later.
